packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -345,9 +349,8 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     clearTimeout(hystersisTimer);
     hystersisTimer = setTimeout(() => {
       hystersisOpen = false;
-    }, 500);
-    // Use 500 ms per https://github.com/reach/reach-ui/blob/3b5319027d763a3082880be887d7a29aee7d3afc/packages/tooltip/src/index.js#L214
-
+    }, 800 + leaveDelay);
+    // Wait x ms, about the same amount of time the Web platform do.
```

Comment:
```suggestion
    // Wait x ms, about the same amount of time the Web platform does.
```

What is "the Web platform"?

Reply:
What about we remove the comment? 😆

Reply:
```suggestion
```

^? 🤷‍♂